lib/tasks/backfill_new_jsonb_columns.rake

```diff
@@ -0,0 +1,30 @@
+desc "Backfill new temp jsonb columns with data from original json colums"
+task backfill_new_jsonb_columns: :environment do
+  AccessLimit.find_each.with_index do |access_limit, index|
+    access_limit.update(temp_users: access_limit.users,
+                        temp_organisations: access_limit.organisations)
+    puts "Updated #{(index + 1)} temp_users and temp_organisations"
+  end
+
+  Edition.find_each.with_index do |edition, index|
+    edition.update(temp_details: edition.details.delete("\u0000"),
+                   temp_routes: edition.routes.delete("\u0000"),
+                   temp_redirects: edition.redirects.delete("\u0000"))
```

Comment:
Did we ever find out where these came from? Does calling `.delete` work on the `Hash`? I imagined we might need to do it on the JSON string.

Reply:
We couldn't figure out how that got there... the .delete didn't work, we'll have to change the rake task